bridge/shared/interfaces/indicator-result.ts

```diff
@@ -5,6 +5,7 @@ export type Target = { criteria: string; targetValue: number; violated: boolean
 export interface IndicatorResult {
   value: {
     value: number;
+    comparedValue: number;
```

Comment:
Does the core have a migration step for `comparedValue`? Else old indicator results will lead to incorrect results or null reference errors. Then I would mark it as optional (`comparedValue?: number`)

Reply:
There is no migration planned, that's why we keep the backwards-compatibility. Made it optional.